includes/blocks/class-wc-block-featured-product.php

```diff
@@ -56,7 +56,7 @@ public static function render( $attributes, $content ) {
 
 		$title = sprintf(
 			'<h2 class="wc-block-featured-product__title">%s</h2>',
-			esc_html( $product->get_title() )
+			$product->get_title()
```

Comment:
Can you add escaping here (the `wp_kses_post` should be good), or add a note if it's escaped elsewhere?

Reply:
Oops, I already had a local commit that added this that I thought I had pushed up. I rebased and pushed up this extra commit now.